docs/GLOSSARY.rst

```diff
@@ -0,0 +1,15 @@
+Glossary
+========
+
+See README.rst for high level information about how to navigate the full documentation.
+
+Glossary of Terms
+-----------------
+
+There are many terms that are highly technical and are not required to use the system, but only to administer it. This glossary is intented to orient users who may be confused about these terms and wish to learn more, but by no means is understanding these necessary for usage of the WSI DeID.
+
+* Docker - An open platform for developing, shipping, and running applications. The WSI DeID is packaged as several Docker containers, which eases installation and distribution of the application. See `Docker <https://docs.docker.com/install/>`_.
```

Comment:
Pedantically, docker calls what we distribute `images` and the running instances of those images `containers`.